src/schemas.js

```diff
@@ -533,3 +548,13 @@ export const nanoContractData = {
     optional: true,
   },
 };
+
+export function bigIntSanitizer(value) {
+  try {
+    return value === undefined || value === null
+      ? value
+      : parseSchema(value, bigIntCoercibleSchema);
+  } catch {
+    return undefined;
+  }
+}
```

Comment:
suggestion: We should add some simple tests for this.

Reply:
Done in https://github.com/HathorNetwork/hathor-wallet-headless/pull/509/commits/9c1caefcf3b04b7b9ba371d35d77643a5d5a0338